### ISSUE TITLE
di-order rule not working !

### ISSUE THREAD

**Issue by anubhavsinha98** [EmmanuelDemey/eslint-plugin-angular#565]
```
hey.thing("ThingName", [
"$http", "$scope", function($scope, $http) {

}
]);
```
Expected Behaviour:
The above code should throw an error of di-order check.
Observed Behaviour:
It's not throwing any error.

**Comment by EmmanuelDemey** [comment]
Hi, 

The `di-order` only check if the parameters are sorted. 

For your issue, your should use the rule `di`. 

You should have a look to this unit test. I think it is exactly the same snippet as yours. 

https://github.com/EmmanuelDemey/eslint-plugin-angular/blob/master/test/di.js#L118
